builds/uninstalling-openshift-builds.adoc

```diff
@@ -0,0 +1,23 @@
+:_content-type: ASSEMBLY
+[id="uninstalling-openshift-builds"]
+= Uninstalling {builds-shortname}
+include::_attributes/common-attributes.adoc[]
+:context: uninstalling-openshift-builds
+
+toc::[]
+
+Cluster administrators can uninstall the {builds-title} Operator by performing the following tasks:
+
+* Delete the Custom Resources (CRs) that were added by default when you installed the {builds-title} Operator.
+* Uninstall the {builds-title} Operator.
+
+Uninstalling only the Operator will not remove the {builds-title} components created by default when the Operator is installed.
+
+include::modules/ob-deleting-the-builds-component-and-custom-resources.adoc[leveloffset=+1]
+
+include::modules/ob-uninstalling-the-builds-operator.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* You can learn more about uninstalling Operators on {product-title} in the link:https://docs.openshift.com/container-platform/latest/operators/admin/olm-deleting-operators-from-cluster.html#olm-deleting-operators-from-a-cluster[deleting Operators from a cluster] section.
```

Comment:
```suggestion
* link:https://docs.openshift.com/container-platform/latest/operators/admin/olm-deleting-operators-from-cluster.html#olm-deleting-operators-from-a-cluster[Deleting Operators from a cluster]
```

Reply:
xref link?

Reply:
same justification given in the first comment